# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsaplayer/gcc44.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur input/flac/FlacPlugin.cpp input/flac/FlacPlugin.cpp
--- ./input/flac/FlacPlugin.cpp	2007-10-29 23:00:16.000000000 +0200
+++ ./input/flac/FlacPlugin.cpp	2009-07-20 00:19:07.000000000 +0300
@@ -239,7 +239,7 @@
     else
     {
 	// use stream name
-	char * fname = strrchr (f->name ().c_str (), '/');
+	const char * fname = strrchr (f->name ().c_str (), '/');
 	if (fname)
 	{
 	    fname++;
@@ -270,7 +270,7 @@
 	if (strncmp(name, "http://", 7) == 0) {
 		return 0.0;
 	}
-	char *ext = strrchr(name, '.');
+	const char *ext = strrchr(name, '.');
 	if (!ext)
 		return 0.0;
 	ext++;
